helix-term/src/health.rs

```diff
@@ -213,6 +211,12 @@ pub fn languages_all() -> std::io::Result<()> {
         }
 
         writeln!(stdout)?;
+
+        cmds.try_for_each(|cmd| {
+            column("", Color::Reset);
+            check_binary(Some(cmd));
+            writeln!(stdout)
+        })?;
```

Comment:
Style-wise I think a for loop is considered cleaner than for_each/try_for_each when possible:

```rust
for cmd in cmds {
    column("", Color::Reset);
    check_binary(Some(cmd));
    writeln!(stdout)?;
}
```

Reply:
Done 👍🏻